secret/store.go

```diff
@@ -132,6 +133,22 @@ func (ss *secretStore) Store(organizationID string, value *CreateSecretRequest)
 
 	log.Debugln("Storing secret:", path)
 
+	// If we are not storing a full TLS secret instead of it's a request to generate one
+	if value.Type == constants.TLSSecretType && len(value.Values) <= 2 {
+		validity := value.Values[constants.TLSValidity]
+		if validity == "" {
+			validity = "8760h" // 1 year
```

Comment:
Can we make the default value configurable?

Reply:
Yes, of course, I will update it.